[2025-10-27T15:48:03.846+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-27T15:48:03.864+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_postgres_etl.extract_from_mysql manual__2025-10-27T15:47:58.119240+00:00 [queued]>
[2025-10-27T15:48:03.870+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_postgres_etl.extract_from_mysql manual__2025-10-27T15:47:58.119240+00:00 [queued]>
[2025-10-27T15:48:03.870+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-27T15:48:03.882+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_from_mysql> on 2025-10-27 15:47:58.119240+00:00
[2025-10-27T15:48:03.887+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=792) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-27T15:48:03.888+0000] {standard_task_runner.py:64} INFO - Started process 794 to run task
[2025-10-27T15:48:03.889+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_postgres_etl', 'extract_from_mysql', 'manual__2025-10-27T15:47:58.119240+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/mysql_to_postgres_etl.py', '--cfg-path', '/tmp/tmpd57afcwx']
[2025-10-27T15:48:03.890+0000] {standard_task_runner.py:91} INFO - Job 71: Subtask extract_from_mysql
[2025-10-27T15:48:03.926+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_to_postgres_etl.extract_from_mysql manual__2025-10-27T15:47:58.119240+00:00 [running]> on host a0076f556b70
[2025-10-27T15:48:03.997+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_postgres_etl' AIRFLOW_CTX_TASK_ID='extract_from_mysql' AIRFLOW_CTX_EXECUTION_DATE='2025-10-27T15:47:58.119240+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-27T15:47:58.119240+00:00'
[2025-10-27T15:48:03.998+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-27T15:48:03.999+0000] {mysql_to_postgres_etl.py:87} INFO - Extracting data for time window: 2025-10-27 14:47:58.119240+00:00 to 2025-10-27 15:47:58.119240+00:00
[2025-10-27T15:48:04.006+0000] {base.py:84} INFO - Using connection ID '***_con' for task execution.
[2025-10-27T15:48:04.018+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:269: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2025-10-27T15:48:04.044+0000] {mysql_to_postgres_etl.py:117} INFO - Extracted 64 rows from MySQL for window: 2025-10-27 14:47:58.119240+00:00 to 2025-10-27 15:47:58.119240+00:00
[2025-10-27T15:48:04.046+0000] {mysql_to_postgres_etl.py:121} INFO - Status distribution in extracted data:
[2025-10-27T15:48:04.046+0000] {mysql_to_postgres_etl.py:123} INFO - - done: 20 records
[2025-10-27T15:48:04.046+0000] {mysql_to_postgres_etl.py:123} INFO - - delivered: 18 records
[2025-10-27T15:48:04.046+0000] {mysql_to_postgres_etl.py:123} INFO - - in_transit: 14 records
[2025-10-27T15:48:04.047+0000] {mysql_to_postgres_etl.py:123} INFO - - created: 12 records
[2025-10-27T15:48:04.070+0000] {python.py:237} INFO - Done. Returned value was:     receipt_id  customer_id  ...          updated_at          deleted_at
0        95344         4341  ... 2025-10-27 14:49:16                 NaT
1           96         4303  ... 2025-10-27 14:50:06 2025-10-27 14:50:06
2        26196         1028  ... 2025-10-27 14:51:11 2025-10-27 14:51:11
3        59202         3608  ... 2025-10-27 14:51:24 2025-10-27 14:51:24
4        92662         4923  ... 2025-10-27 14:52:58                 NaT
..         ...          ...  ...                 ...                 ...
59       67953          901  ... 2025-10-27 15:45:38                 NaT
60       63858         4612  ... 2025-10-27 15:45:54                 NaT
61       48366         2508  ... 2025-10-27 15:46:37                 NaT
62       32707         1970  ... 2025-10-27 15:46:41 2025-10-27 15:46:41
63        5092         1556  ... 2025-10-27 15:47:53                 NaT

[64 rows x 8 columns]
[2025-10-27T15:48:04.088+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-27T15:48:04.107+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_to_postgres_etl, task_id=extract_from_mysql, run_id=manual__2025-10-27T15:47:58.119240+00:00, execution_date=20251027T154758, start_date=20251027T154803, end_date=20251027T154804
[2025-10-27T15:48:04.150+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-27T15:48:04.174+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-27T15:48:04.175+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
